### ISSUE TITLE
Adding feet key points to the model for 3D conversions 

### ISSUE THREAD

**Issue by kaurrachneet6** [ArashHosseini/3d-pose-baseline#8]
Hello,
I want to use the pretrained model that you provide to convert my estimated 2D coordinates from openpose body 25 model, where the toes and heels of both feet are also included as predicted 2D coordinates, into 3D coordinates. I want to ask 2 questions:
1. Is it possible to do so, since the pretrained model was not tuned for these extra toe and heel coordinates. In fact, I would like to elaborate that I want to only use 12 coordinates in total, where 6 are feet coordinates (toes and heels) and 6 are other body coordinates that your model is trained on. So, I can append 5 body coordinates as zeros to make the input size 17 (same as your network's input and output row size) and then run your pretrained model. Would the pretrained model still be valid to use in this case. 
2. If it is still feasible to use pretrained model for predicting these extra feet coordinates in 3D as well, what changes are required in the openpose_3dpose_sandbox.py or any other files as well, to give 2D input with extra 6 coordinates (2 toe and 1 heel from each foot), and get 3D output with these extra 6 coordinates estimates as well. 

I want to emphasize that I do not have any ground truth labels, hence I must use the pretrained model only without any fine-tuning too. Thanks.
